package.json

```diff
@@ -17,27 +17,28 @@
     "version:dependencies": "node ./scripts/update-version.js && npm install",
     "langium:generate": "npm run langium:generate --workspace=langium --workspace=examples/domainmodel --workspace=examples/arithmetics --workspace=examples/statemachine --workspace=examples/requirements",
     "dev-build": "npm run dev-clean && npm install && npm link ./packages/langium && npm link ./packages/langium-cli && npm link ./packages/generator-langium",
-    "dev-clean": "shx rm -rf packages/**/node_modules && npm uninstall -g langium-workspaces langium-cli generator-langium langium && npm unlink langium-workspaces langium-cli generator-langium langium"
+    "dev-clean": "shx rm -rf packages/**/node_modules && npm uninstall -g langium-workspaces langium-cli generator-langium langium && npm unlink langium-workspaces langium-cli generator-langium langium",
+    "reset:repo": "git clean -f -X -d"
```

Comment:
I don't see much value in this: `git clean -fdX` is shorter than `npm run reset:repo` and I need to look into package.json to understand what it does.

Reply:
🙂 `npm run --list` helps and I find `npm run reset:repo` more easy to remember, but that is personal preference. Also, if you ever need exclusion this will be more handy. 😉 Your call.

Reply:
Ok :-)